### DISCUSSION TITLE
how to know which device is connected

### DISCUSSION THREAD

**Discussion by MC-dusk** [LizardByte/community#507] [Sunshine] [answered]
How to know which devices are connected to the host which is running Sunshine?
I run sunshine in the background, but I can just see a toast telling me that a device is connecting. I don't know which device it is. Is it my device or something else? What is the name and ip of the device? Can I somehow get the infomation of the remote device?
Intuitively speaking, there should be a list of linked devices in the web ui, where I can see all these remote device and control the connection. Or my PC may be connected by someone else and I do not realize cause there can be multi devices to link at the same time.

**Comment by ReenigneArcher** [comment, marked as answer]
In beta versions, there is a list of paired devices, but there is not indicator of which device is connected. You might be able to get some of the info out of the log.
